connection_test.go

```diff
@@ -12,7 +12,7 @@ func TestQuery(t *testing.T) {
 	db, err := sql.Open("maxcompute", cfg4test.FormatDSN())
 	a.NoError(err)
 
-	const stmt = `SELECT * FROM gomaxcompute_test LIMIT 1;`
+	const stmt = `SELECT * FROM gomaxcompute_test LIMIT 1`
```

Comment:
How about testing this in a separate test?

Reply:
You're right!